include/fam/fam.h

```diff
@@ -114,6 +128,11 @@ typedef enum {
     RAID5
 } Fam_Redundancy_Level;
 
+typedef struct {
```

Comment:
Might need to properly format the code.
Use clang-format tool to follow the coding convention and make appropriate changes.

Reply:
addressed